trin-core/src/portalnet/storage.rs

```diff
@@ -689,7 +696,9 @@ impl PortalStorage {
 
     /// Helper function for opening a RocksDB connection for the radius-constrained db.
     pub fn setup_rocksdb(node_id: NodeId) -> Result<rocksdb::DB, ContentStoreError> {
-        let mut data_path: PathBuf = get_data_dir(node_id);
+        let mut data_path: PathBuf = get_data_dir(node_id).map_err(|err| {
+            ContentStoreError::Database(format!("unable to get data dir: {:?}", err))
```

Comment:
```suggestion
            ContentStoreError::Database(format!("unable to get data dir for rocksdb: {:?}", err))
```

for error uniqueness